README.md

```diff
@@ -5,7 +5,65 @@ This repository contains the code to generate various versions of the Datadog Ag
 * https://s3.amazonaws.com/dd-agent/scripts/install_script_agent6.sh to install Agent 6
 * https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh to install Agent 7
 
-Usage instructions for the install script are in the [Datadog App](https://app.datadoghq.com/account/settings#agent/overview).
+
+## Agent install script usage
+Basic usage instructions for the Agent install script are in the [Datadog App](https://app.datadoghq.com/account/settings#agent/overview).
+
+### Agent configuration options
+The install script allows installation of different flavors of the Agent binaries and allows you to set some configuration options automatically. We list them below with their different possible values. 
+
+> [!IMPORTANT]
+> All variables are optional except `DD_API_KEY`, which is required unless `DD_UPGRADE` (upgrade from datadog-agent 5) is set.
+
+> [!WARNING]
+> The install script input options are only considered at initial installation and they don't overwrite pre-existing configuration files
+
+| Variable | Description|
+|-|-|
+|`DD_AGENT_FLAVOR`|The Agent binary to install. Possible values are `datadog-agent`(default), `datadog-iot-agent`, `datadog-dogstatsd`, `datadog-fips-proxy`, `datadog-heroku-agent`.|
+|`DD_API_KEY`|The application key to access Datadog's programatic API.|
+|`DD_SITE`|The site of the Datadog intake to send Agent data to. For example, `datadoghq.com`. For more information on Datadog sites, see [Getting Started with Datadog Sites](https://docs.datadoghq.com/getting_started/site/)|
+|`DD_URL`|The host of the Datadog intake server to send metrics to. For example, `https://app.datadoghq.com`. Only set this option if you need the Agent to send metrics to a custom URL: it overrides the site setting defined in `site`. It does not affect APM, Logs or Live Process intake which have their own `*_dd_url` settings.|
+|`DD_HOSTNAME`|Force the hostname name.|
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏"hostname name" sounds awkward, perhaps "hostname value".
```suggestion
|`DD_HOSTNAME`|Force the hostname value.|
```